.vib/mongodb-sharded/ginkgo/mongodb_suite_test.go

```diff
@@ -0,0 +1,21 @@
+package mongodb_sharded_test
+
+import (
+	"flag"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var kubeconfig = flag.String("kubeconfig", "", "absolute path to the kubeconfig file")
+var releaseName = flag.String("name", "", "name of the primary statefulset")
+var namespace = flag.String("namespace", "", "namespace where the application is running")
+var username = flag.String("username", "", "database user")
+var password = flag.String("password", "", "database password for username")
+var shards = flag.Int("shards", 3, "number of shards")
```

Comment:
I'd declare all the variables at once:

```go
var (
	kubeconfig = flag.String("kubeconfig", "", "absolute path to the kubeconfig file")
	...
)
```

Reply:
Also, in order to avoid dealing with pointers in your code, I'd recommend using the Var() functions and an `init()` function. Sth like this:

```go
var (
	kubeconfig string
        ...
        shards int
)

func init() {
	flag.StringVar(&kubeconfig, "kubeconfig", "", "absolute path to the kubeconfig file")
	...
	flag.IntVar(&shards, "shards", 3, "number of shards")
}
````

Reply:
Fixed here https://github.com/bitnami/charts/pull/18777/commits/3a0c68d0273b7998f569b6456b92b3bd684c3106